tests/helpers.py

```diff
@@ -460,3 +460,30 @@ def get_cephfs_name():
     )
     result = CFS.get()
     return result['items'][0].get('metadata').get('name')
+
+
+def get_pv_for_pvc(pvc_obj):
```

Comment:
in PVC class under pvc.py there's a property called backed_pv.
please move this implementation to there

Reply:
@RazTamir ,
But in tess/fixture.py pvc is created through OCS object not PVC object. so still I can't access backed_pv.